plugins/reports/counter/classes/reports/AR1_r4_1.inc.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+/**
+ * @file plugins/generic/sushiLite/classes/reports/AR1_r4_1.inc.php
+ *
+ * Copyright (c) 2014 University of Pittsburgh
+ * Distributed under the GNU GPL v2 or later. For full terms see the file docs/COPYING.
+ *
+ * @class AR1_r4_1
+ * @ingroup plugins_reports_counter
+ *
+ * @brief Article Report 1, release 4.1
+ */
+
+import('plugins.reports.counter.classes.releases.CounterRelease4_1');
+
+class AR1_r4_1 extends CounterRelease4_1 {
+	
+	/**
+	 * Constructor
+	 */
+	function AR1_r4_1($parentPluginCategory, $parentPluginName) {
+		parent::CounterRelease4_1($parentPluginCategory, $parentPluginName);
+	}
+
+	/**
+	 * Get the report title
+	 * @return $string
+	 */
+	function getTitle() {
+		return __('plugins.reports.counter.ar1.title');
+	}
+
+	/*
+	 * Convert an OJS metrics request to a COUNTER ReportItem
+	 * @param $columns string|array column (aggregation level) selection
+	 * @param $filters array report-level filter selection
+	 * @param $orderBy array order criteria
+	 * @param $range null|DBResultRange paging specification
+	 * @see ReportPlugin::getMetrics for more details
+	 * @return object COUNTER\ReportItem
+	 */
+	function getMetrics($columns = array(), $filters = array(), $orderBy = array(), $range = null) {
```

Comment:
I think it would be better to call this method getReportItems(). getMetrics is already used in other places to get the raw stats metrics, and this is not the case here.